templates/nginx-omero.conf.j2

```diff
@@ -4,8 +4,27 @@ server {
 }
 
 server {
+    listen 443 ssl;
     server_name $hostname;
 
+    ssl_certificate {{ ssl_certificate_bundled_path }};
+    ssl_certificate_key {{ ssl_certificate_key_path }};
+    ssl_protocols   TLSv1 TLSv1.1 TLSv1.2;
```

Comment:
While we're at it should we just drop at least `TLSv1`? (Or do we not actually support it?)

Reply:
The current list matches the latest Nginx default: http://nginx.org/en/docs/http/ngx_http_ssl_module.html#ssl_protocols